src/views/Maintenance.vue

```diff
@@ -12,25 +12,21 @@ import { EventBus } from '../utils/eventBus';
 export default {
   mounted() {
     const interval = setInterval(() => EventBus.$emit('reloadData'), 10 * 1000);
```

Comment:
Actually, it calls only a `reloadData` method of App component. I think it should be extracted to the store and called explicitly.